bambi/results.py

```diff
@@ -1,13 +1,23 @@
-from abc import abstractmethod, ABCMeta
-from bambi.external.six import string_types
-import arviz as az
 import re
-import warnings
-import pandas as pd
+from abc import ABCMeta, abstractmethod
+
+import arviz as az
 import numpy as np
-import matplotlib.pyplot as plt
+import pandas as pd
+from bambi.external.six import string_types
+
 from .utils import listify
 
+try:
```

Comment:
I removed all this code in  #169

Reply:
(specifically, lines 11-19 can be deleted)